test/spec/modules/lotamePanoramaIdSystem_spec.js

```diff
@@ -440,10 +440,95 @@ describe('LotameId', function() {
     });
   });
 
-  it('should retrieve the id when decode is called', function() {
-    var id = lotamePanoramaIdSubmodule.decode('1234');
-    expect(id).to.be.eql({
-      'lotamePanoramaId': '1234'
+  describe('when gdpr applies and falls back to eupubconsent cookie', function () {
```

Comment:
The fallback to the cookies should have tests where consentdata and gdpr applies aren't populated as those are both populated by the cmp. The way the code is currently written will have no practical effect.